src/Features/LanguageServer/Protocol/LspServices/ExportRoslynLspServiceFactoryAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler;
+
+/// <summary>
+/// Defines an easy to use subclass for <see cref="ExportLspServiceFactoryAttribute"/> with the roslyn languages contract name.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class, AllowMultiple = false), MetadataAttribute]
+internal class ExportRoslynLspServiceFactoryAttribute : ExportLspServiceFactoryAttribute
```

Comment:
❔ Why `Roslyn` in the name of this attribute?

Reply:
this was to differentiate between the c#/vb ones vs. Xaml vs. JS.  I can potentially rename to CSharpVisualBasicXXX though.